extra/notebooks.ipynb

```diff
@@ -0,0 +1,364 @@
+    "\n",
+    "If your environment does not include Jupyter, you can install it by running the following command in your conda environment:\n",
+    "\n",
+    "```console\n",
+    "conda install notebook\n",
+    "```\n",
+    "\n",
+    "### Starting Server\n",
+    "\n",
+    "With Jupyter installed, let's start the Notebook server by opening your terminal application in the home folder of the project repository and running the following command:\n",
+    "\n",
+    "```console\n",
+    "jupyter notebook\n",
+    "```\n",
+    "\n",
+    "This will start up Jupyter, and your default browser should start (or open a new tab) to the following URL: [http://localhost:8888/tree](http://localhost:8888/tree). The opened page should look like this:\n",
+    "\n",
+    "![Jupyter Notebook web app](../static/images/initial_notebook_screen.png \"Jupyter Notebook web app\")\n",
+    "\n",
+    "> ✏️ Alternatively, you can use an [IDE](ide-overview.md) to interact with the server\n",
+    "\n",
+    "## Running your first notebook\n",
+    "\n",
+    "Let's familiarize ourselves a bit with the Jupyter Notebook web app and its interface.\n",
+    "\n",
+    "### Creating a Notebook\n",
+    "\n",
+    "Browse to the folder in which you would like to create your first notebook, click the \"New\" drop-down button in the top-right and select \"Python 3\":\n",
+    "\n",
+    "![The new Notebook menu](../static/images/new_notebook_menu.webp \"The new Notebook menu\")\n",
+    "\n",
+    "This should create a new notebook in the selected folder similar to this:\n",
+    "\n",
+    "![A new Notebook](../static/images/new_notebook.png \"A new Notebook\")\n",
+    "\n",
+    "You might notice the word _Untitled_ at the top of the page. It is the title of the page and the name of your notebook. As a good practice, change the name to something more descriptive. We use [Snake case](https://en.wikipedia.org/wiki/Snake_case) for variable names, function names, method names, and module or package (i.e., file) names. Move your mouse over the word _Untitled_ and click on the text. You should now see an in-browser dialog titled _Rename Notebook_, so do.\n",
+    "\n",
+    "> ✏️ Each time you create a new notebook, a new _.ipynb_ file will be created. Each _.ipynb_ file is a text file that describes your notebook's contents in a _JSON_ format.\n",
+    "\n",
+    "### Executing cells\n",
+    "\n",
+    "A Notebook’s cell defaults to using code whenever you create one, and that cell uses the kernel you chose when you started your Notebook.\n",
+    "\n",
+    "Thus, to verify that everything is working as it should, you can add some Python code to the cell and try running its contents. For example, try the following code that will **plot some data**:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
```

Comment:
This is your first code cell, but execution count is equal to 7. When publishing notebooks, it is always a good practice to restart kernel and run all cells after finishing your work. The reason is simple - to ensure your notebook is reproducible. It may occur you define a variable, and then you delete a cell with that variable, so your notebook is not reproducible anymore.